reframe/frontend/ci.py

```diff
@@ -11,7 +11,7 @@
 import reframe.core.runtime as runtime
 
 
-def _emit_gitlab_pipeline(testcases):
+def _emit_gitlab_pipeline(testcases, child_pipeline_opts):
```

Comment:
```suggestion
def _emit_gitlab_pipeline(testcases, child_pipeline_opts=None):
```

And avoid having to change the unit test.